src/mame/drivers/savquest.cpp

```diff
@@ -831,22 +907,27 @@ void savquest_state::savquest(machine_config &config)
 	pcat_common(config);
 	DS12885(config.replace(), "rtc");
 
-	pci_bus_legacy_device &pcibus(PCI_BUS_LEGACY(config, "pcibus", 0, 0));
+	pci_bus_legacy_device &pcibus(PCI_BUS_LEGACY(config, "pcibus", 34_MHz_XTAL, 0)); // PII PCI bus clock 33/34/37/40.1 MHz
```

Comment:
XTAL implies that there's a 34MHz quartz crystal. Is that really the case or did you make a guess? Same for the 9MHz ISA one below.

Reply:
Hmmm for some reason the reference documentation got lost in commit. Fixed that now...

The timings come from this document specification of the Pentium II motherboard:
https://web.archive.org/web/20120630212356/http://www.motherboards.org/files/manuals/42/MB-P6F91i.pdf

Page 7:

Speed System bus clock 66/68/75/83MHz
System bus clock 100/103/112/133MHz
AGP clock 66/68/75/83 MHz
PCI bus clock 33/34/37/40.1 MHz
ISA bus clock 8.33~9.35 MHz

I just took the closest XTAL values that matched these.

Reply:
Yeah, my point is, they are not 2 separate XTALS.
That doc says bus clock is configurable in the BIOS. Let's assume this Pentium 2 450MHz has a bus clock of 100MHz. Then PCI clock is 100'000'000 / 3, and ISA clock is 100'000'000 / 12.